app/controllers/application_controller.rb

```diff
@@ -1,24 +1,34 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
-    # Prevent CSRF attacks by raising an exception.
-    # For APIs, you may want to use :null_session instead.
-    protect_from_forgery with: :exception
-    before_action :configure_permitted_parameters, if: :devise_controller?
+  include Pundit
```

Comment:
This is one of the two actual changes in this file. Most of the other changes are formatting (this file was auto-generated, so when I touched it Prettier formatted it for the first time).